app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,159 @@
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="10dp"
+        android:orientation="vertical"
+        tools:ignore="MissingConstraints">
+
+        <TextView
+            style="@style/tittleStyle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="5dp"
+            android:layout_marginLeft="5dp"
+            android:layout_marginTop="20dp"
+            android:text="@string/textViewTittle">
+
+        </TextView>
+
+        <TextView
+            style="@style/tittleStyle.item"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="5dp"
+            android:layout_marginLeft="5dp"
+            android:text="@string/textViewitem1">
+
+        </TextView>
+
+        <TextView
+            style="@style/tittleStyle.item"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+
+            android:layout_marginStart="5dp"
+            android:layout_marginLeft="5dp"
+            android:text="@string/textViewitem2">
+
+        </TextView>
+
+        <TextView
+            style="@style/tittleStyle.item"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+
+            android:layout_marginStart="5dp"
+            android:layout_marginLeft="5dp"
+            android:text="@string/textViewitem3">
+
+        </TextView>
+
+    </LinearLayout>
+
+    <ImageView
+        android:id="@+id/wifiImage"
+        android:layout_width="match_parent"
+        android:layout_height="260dp"
+        android:layout_marginTop="16dp"
+        android:background="@drawable/level_list_drawable"
+        app:layout_constraintTop_toBottomOf="@+id/textViewLayout"
+        tools:layout_editor_absoluteX="0dp">
+
+    </ImageView>
+
+    <LinearLayout
+
+        android:id="@+id/imageButtonLayout"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_margin="20dp"
+        android:orientation="horizontal"
+        android:weightSum="3"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/wifiImage">
+
+        <ImageButton
+            android:id="@+id/plus_image_button"
+            android:layout_width="@dimen/ImageButtonWidth"
+            android:layout_height="@dimen/ImageButtonHeight"
+            android:layout_marginLeft="@dimen/ImageButtonMarginLeft"
+            android:layout_weight="1"
+            android:background="@drawable/plus_background"
+            android:padding="10dp">
+
+        </ImageButton>
+
+        <ImageButton
+            android:id="@+id/minus_image_button"
```

Comment:
As mentioned in the assignment doc create below styles:
1. Base style, which includes all common attribute for plus/minus and off button.
2. Plus button style, which includes the plus button specific attribute. Inherit this style from the Base style
3. Minus button style, which includes the minus button specific attributes. Inherit this style from Base style.
4. Off button style, which includes the Off button specific attributes. Inherit this style from Base style.